examples/get_schedule_block_info.py

```diff
@@ -79,6 +79,14 @@ def main():
         #              u'outcome': u'UNKNOWN',
         #              u'data_quality': None}
 
+    cb_ids = ['1555494792.25']
+    # cb_ids = yield portal_client.schedule_blocks_completed()
```

Comment:
Was this line here for testing or is it something that got left out?

Reply:
I thought I will need to add a method in portal_client, namely `schedule_blocks_completed`, but at the standup I realised I actually don't need to. Will remove this line now.